### DISCUSSION TITLE
Get correct video data on recent videos scan mode

### DISCUSSION THREAD

**Discussion by DinhHuy2010** [ThioJoe/YT-Spammer-Purge#791] [Ideas]
I think @ThioJoe should get the all Video ID and validate them on scan recent videos so to get correct video data.

**Comment by Firecul** [comment]
Can you explain what you mean please, I don't get what you are saying?

**Reply by DinhHuy2010** [reply]
On recent videos mode, while fetching the video data. Instead of direct get the data from response, take the video ID and query to API it and get the data from the only response so like video title will not have weird characters like this.

![Screenshot 2022-04-10 120507](https://user-images.githubusercontent.com/95196459/162602255-d03f23ef-97e5-4d9c-80bc-c92751fb4d37.png)

**Reply by Kurdakov** [reply]
it's 
import html
html.unescape

which will normalize strings

**Reply by DinhHuy2010** [reply]
ok

**Reply by ThioJoe** [reply]
Hm yea I've noticed that too, shouldn't be to hard to fix

**Comment by ThioJoe** [comment]
This should be fixed with the latest releases
